brianlong/www.validators.app
# 186442321 - add sitemap

#### What's this PR do?
- Applies sitemap and robots.txt

#### How should this be manually tested?
- instructions designed both for development and staging platforms
- run `rake sitemap:refresh:no_ping` in order to create sitemaps. The output should include information about compressed sitemap(s) files
- run server and enter /robots.txt. Verify disallowed paths (pages where the robot should not process).
- download the sitemap file over the path /sitemap.xml.gz. It will include an information about indexed sitemaps (sitemap1.xml.gz, sitemap2.xml.gz and so on). Download those files and verify whether they have links to the application sites (validators, data_centers, vote_accounts, explorer_stake_accounts and others mentioned in config/sitemap.rb file).

#### What are the relevant tickets?
[- URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/186442321)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
